apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.Tests/CodeHealthTest.cs

```diff
@@ -22,7 +22,8 @@ public class CodeHealthTest
         [Fact]
         public void PrivateFields()
         {
-            CodeHealthTester.AssertAllFieldsPrivate(typeof(BigQueryClient));
+            // BigQueryProviderFactory must have a public Instance field to work properly as its loaded and instantiated via reflection.
```

Comment:
https://docs.microsoft.com/en-us/dotnet/framework/data/adonet/obtaining-a-dbproviderfactory

psuedo magic mechanisms for obtaining the provider factory where you only specify its type information in your config.  Why they didnt opt to simply specify the public member (which could be a property) I have no idea -- but its designed to automatically look for a public static field when building the providers from config.